manifests/ruby.yml

```diff
@@ -7,18 +7,18 @@ tests/:
   appsec/:
     api_security/:
       test_schemas.py:
-        Test_Schema_Request_Body: missing_feature
-        Test_Schema_Request_Cookies: missing_feature
-        Test_Schema_Request_Headers: missing_feature
-        Test_Schema_Request_Path_Parameters: missing_feature
-        Test_Schema_Request_Query_Parameters: missing_feature
+        Test_Schema_Request_Body: v1.15.0
+        Test_Schema_Request_Cookies: v1.15.0
+        Test_Schema_Request_Headers: v1.15.0
+        Test_Schema_Request_Path_Parameters: v1.15.0
```

Comment:
Failing on rack

Reply:
Ohh crap. Rack does not support the feature. Will add that in the specs